pointcept/engines/hooks/evaluator.py

```diff
@@ -176,7 +183,12 @@ def eval(self):
                 m_iou, m_acc, all_acc
             )
         )
+        current_epoch = self.trainer.epoch + 1
         for i in range(self.trainer.cfg.data.num_classes):
+            name = self.trainer.cfg.data.names[i]
+            self.trainer.writer.add_scalar(
+                f"val/cls_{i}-{name} Iou", iou_class[i], current_epoch
```

Comment:
Additional logging class/iou.

Reply:
I think these are some unnecessary details which might confuse our user, especially for tensorboard/wandb, especially when we have hundreds of categories